src/containers/SortingView.tsx

```diff
@@ -269,12 +287,21 @@ const mapStateToProps: MapStateToProps<StateProps, OwnProps, RootState> = (state
 
 const mapDispatchToProps: MapDispatchToProps<DispatchProps, OwnProps> = (dispatch: Dispatch<RootAction>, ownProps: OwnProps) => {
   const curationDispatch = (a: SortingCurationAction) => {
-    if (a.type === 'AddLabel') {
+    if (a.type === 'SetCuration') {
+      dispatch({type: 'SET_CURATION', sortingId: ownProps.sortingId, curation: a.curation, persistKey: 'sortings'})
+    }
+    else if (a.type === 'AddLabel') {
       dispatch({type: 'ADD_UNIT_LABEL', sortingId: ownProps.sortingId, unitId: a.unitId, label: a.label, persistKey: 'sortings'})
     }
     else if (a.type === 'RemoveLabel') {
       dispatch({type: 'REMOVE_UNIT_LABEL', sortingId: ownProps.sortingId, unitId: a.unitId, label: a.label, persistKey: 'sortings'})
     }
+    else if (a.type === 'MergeUnits') {
```

Comment:
I don't know why these changes are showing up in this PR... this should already be merged into master. Hmmm.